GoogleSignInSwift.podspec

```diff
@@ -0,0 +1,27 @@
+Pod::Spec.new do |s|
+  s.name = 'GoogleSignInSwift'
+  s.version = '6.2.0'
+  s.swift_version = '4.0'
+  s.summary = 'Adds Swift-focused support for Google Sign In.'
+  s.description = 'Adds Swift-focused support for Google Sign In.'
```

Comment:
Looks like I broke your pod lib lint here.  How about we try:

"Additional Swift support for the Google Sign-In SDK."